app/root.tsx

```diff
@@ -1,3 +1,4 @@
+import {cssBundleHref} from '@remix-run/css-bundle/dist/index'
```

Comment:
Hmmm... is this really necessary? Can we not do this:

```suggestion
import {cssBundleHref} from '@remix-run/css-bundle'
```

That seems to work in the Epic Stack.